src/nxdoc/nuxeo-server/administration/variables-and-configuration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -386,11 +391,15 @@ _Default value:_ {{! multiexcerpt name='org.nuxeo.automation.trace.printable-def
 *
 {{! /multiexcerpt}}
 
-#### `templateName.target`
+#### `templateName.include` - only available in template properties
+
+Templates to inlcude prior to the one being included.
```

Comment:
```suggestion
Templates to include prior to the ones being included.
```

Reply:
Keeping one as it is relative to the `templateName`.